setup.py

```diff
@@ -73,12 +107,13 @@ def build_extension(self, ext):
 
         cmake_args = [
             "-DCMAKE_LIBRARY_OUTPUT_DIRECTORY={}".format(extdir),
+            "-DCMAKE_RUNTIME_OUTPUT_DIRECTORY={}".format(bindir),
             "-DPython_EXECUTABLE={}".format(sys.executable),
             # Not used on MSVC, but no harm
             "-DCMAKE_BUILD_TYPE={}".format(cfg),
-            "-DBUILD_SHARED_LIBS=OFF",
+            "-DBUILD_SHARED_LIBS=ON",
             "-DOCIO_BUILD_DOCS=ON",
```

Comment:
Unrelated to this, but is there a reason why the docs are built here? It makes it so that to build a wheel, you need doxygen and sphinx installed.

Reply:
This is used for the docstring, I made this optional (see also this issue #1744).

Reply:
I see. I'm not too sure if I understand what you mean. Anyway, my comment is unrelated to this PR. I just think that in general, building a wheel shouldn't involve documentation tools.

Reply:
The documentation written in the C++ header will end up in the generated docstring for the Python wheel, developer can use help() to see the documentation, or the IDE can show it directly.